src/Files.App/UserControls/Menus/FileTagsContextMenu.cs

```diff
@@ -46,7 +37,8 @@ public FileTagsContextMenu(IEnumerable<ListedItem> selectedItems)
 
 			SelectedItems = selectedItems;
 
-			Opening += Item_Opening;
+			if (SelectedItems is not null)
```

Comment:
Since the event has a time lag before it occurs, it would be better to do a null check in the event.